packages/react/src/components/Link/Link.tsx

```diff
@@ -0,0 +1,31 @@
+import type { AnchorHTMLAttributes, ReactNode } from 'react';
+import React, { forwardRef } from 'react';
+import cn from 'classnames';
+
+import classes from './Link.module.css';
+
+export type LinkProps = {
+  /** The content to display inside the link. */
+  children: ReactNode;
+
+  /** Custom class name for the link. This will be appended to the design system class names. */
+  className?: string;
```

Comment:
Maybe remove this description? We don't have if for `className` on all our other components. Would say its a given?

Reply:
Is it though? I think it's relevant to inform that the class name is appended and will not override the default.

Reply:
Are there any usecases it will override? Appending className is default behaviour for all our components, so not sure we need to document default behaviour 🤔

Reply:
New users are not aware of the default behaviour.